### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by harapalb1** [igvteam/igv#1013]
Hello.

I have recently used IGV on some chromosomes, and on some segments with lengths of 190000 bp and more, in which _**the readings and coverage in the CRAM file suddenly disappear, although the data in the VCF file remain visible.**_ An error message is also displayed:

_Error - possible sequence mismatch (wrong reference for this file): htsjdk.samtools.cram.CRAMException: Reference MD5 failed to validate against 9616e942ea0176c32537840b45a66348. In other places the message is the same but the code of numbers and letters at the end differs._

Is this phenomenon caused by errors in the CRAM file?

Thank you in advance.

**Comment by harapalb1** [comment]
![k5](https://user-images.githubusercontent.com/88622881/128636836-040e81ec-7d7f-49ab-80eb-d89132cb98ba.jpg)

**Comment by jrobinso** [comment]
It means you are attempting to view the CRAM file with a reference sequence that differs from that on which the CRAM file was created.   For example the hg38 reference sequence that the 1KG used differs from hg38 sequence distributed from UCSC in some locations.

**Comment by harapalb1** [comment]
> 
> 
> It means you are attempting to view the CRAM file with a reference sequence that differs from that on which the CRAM file was created. For example the hg38 reference sequence that the 1KG used differs from hg38 sequence distributed from UCSC in some locations.

Thank you.

My level of knowledge is low in this field. Is it possible for a WGS result to have the CRAM file built after different references?

**Comment by jrobinso** [comment]
@harapalb1 Yes, and that's whats probably happened here.  IGV provides 2 hg38 references, I would suggest retrying with the genome labeled. "Human (hg38 1kg/GATK)", assuming you are using "Human (hg38)" now.  The sequence differences between these assemblies are negligible,  IIRC ambiguity codes in stead of "NNNNN..." in some locations, but any mismatch will invalidate a CRAM file.    Its unfortunate and there's really know way for an end user to know what was used other than asking the create of the CRAM.    If that doesn't resolve your problem reply here, the next step would be to examine the header of the CRAM file with samtools.
